cmd/platform/export.go

```diff
@@ -69,7 +77,19 @@ func exportRunE(cmd *cobra.Command, args []string) error {
 
 func initGeneralExportFlags(cmd *cobra.Command) {
 	cmd.Flags().AddFlag(options.PlatformExportExportFormatOption.Flag)
+	// auto-completion
+	err := cmd.RegisterFlagCompletionFunc("format", autocompletion_export_flags.Format)
```

Comment:
Perhaps it would be better to organize under a single `autocomplete` package instead of individual `autocompletion_export_flags` packages for example.

Something like `autocompletion.PlatformExportFormatFunc`